massa-grpc/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "massa_grpc"
```

Comment:
i think this module should be in massa-api

Reply:
At the moment, we have an existing module with this name.